kakmerge

```diff
@@ -15,9 +15,12 @@ readonly KAKMERGE_CMDS='
         Search for the contents of the main selection in all clients (local, base, remote, merged)
     } %{ eval -draft %{
         exec -save-regs "" <space> *
-        exec -client local n
-        exec -client base n
-        exec -client remote n
+        try %{ exec -client local n }
+        try %{ exec -client base n }
+        try %{ exec -client remote n }
```

Comment:
Wrapping in `try` allows to select a line that only exists in `theirs` and use `kakmerge-search` to find it.